spark-jobs/src/main/scala/za/co/absa/enceladus/common/Repartitioner.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.common
+
+import org.apache.spark.sql.DataFrame
+import za.co.absa.enceladus.common.config.CommonConfConstants
+import za.co.absa.enceladus.utils.config.ConfigReader
+import za.co.absa.spark.partition.sizing.DataFramePartitioner.DataFrameFunctions
+import za.co.absa.spark.partition.sizing.sizer._
+import za.co.absa.spark.partition.sizing.types.DataTypeSizes
+import za.co.absa.spark.partition.sizing.types.DataTypeSizes.DefaultDataTypeSizes
+
+class Repartitioner(configReader: ConfigReader) {
```

Comment:
I know I have been hands-off on this, but can anybody tell me why this is part of Enceladus, but seems as it should/could be part of the spark-partition-sizing library.

Reply:
Well, the library is supposed to provide the functions themselves. This class is supposed to accomodate the functions with the configs

Reply:
I like the class as an isolation level to gather and provide (re-)partitioning settings separated from the main flow.
But I think the current implementation is too ambitious. See my other comments.